ftdc/custom_format.go

```diff
@@ -152,77 +241,66 @@ func getFieldsForItem(item any) []string {
 // For correctness, it must be the case that the `mapOrder` and `fieldOrder` are consistent. I.e: if
 // the `mapOrder` is `A` then `B`, the `fieldOrder` must list all of the fields of `A` first,
 // followed by all the fields of `B`.
-func getSchema(data map[string]any) *schema {
+func getSchema(data map[string]any) (*schema, *schemaError) {
```

Comment:
Would've thought this'd return `(*schema, error)` instead of a pointer to the error type, but seems fine.

Reply:
You're right I probably didn't need to do it this way. I don't recall thinking about this, so I probably just wrote what came to mind.